metis-authentication/metis-authentication-rest/src/main/java/eu/europeana/metis/authentication/rest/AuthenticationController.java

```diff
@@ -253,6 +254,34 @@ public void updateUserToMakeAdmin(@RequestHeader("Authorization") String authori
     }
   }
 
+  /**
+   * Get a user using a user identifier.
+   * <p>POST method is used to pass the user identifier through the body</p>
+   *
+   * @param authorization the String provided by an HTTP Authorization header <p> The expected input
+   * should follow the rule Bearer accessTokenHere </p>
+   * @param userIdParameter the class that contains the userId parameter to act upon
+   * @return the metis user with sensitive information removed
+   * @throws GenericMetisException which can be one of:
+   * <ul>
+   * <li>{@link UserUnauthorizedException} if the authorization header is un-parsable or the user
+   * cannot be authenticated.</li>
+   * </ul>
+   */
+  @RequestMapping(value = RestEndpoints.AUTHENTICATION_USER_BY_USER_ID, method = RequestMethod.POST, produces = {
+      MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_XML_VALUE})
+  @ResponseBody
+  public MetisUser getUserByUserId(@RequestHeader("Authorization") String authorization,
+      @RequestBody UserIdParameter userIdParameter) throws GenericMetisException {
```

Comment:
Can the user ID not be a url parameter (REST-like)? Is this considered too sensitive to put in a query?

Reply:
Well I followed this practice since we did the same for another call that had as a query parameter the user email.